anni/Cargo.toml

```diff
@@ -36,13 +36,15 @@ anyhow = "1.0"
 cuna = "0.6.3"
 id3 = "1"
 anni-vgmdb = "0.3.0"
+musicbrainz_rs = { git = "https://github.com/snylonue/musicbrainz_rs.git", branch = "release_with_artist_credit"}
```

Comment:
publish to crate.io?

Reply:
This is a patched version. I will open a pr later.

Reply:
If the pull request can not be merged by upstream due to inactivity, just fork this repository to this organization. I've sent an invitation to you.

Reply:
done